Formula/chunkwm.rb

```diff
@@ -0,0 +1,108 @@
+class Chunkwm < Formula
+  desc "Tiling window manager for macOS based on plugin architecture"
+  homepage "https://github.com/koekeishiya/chunkwm"
+  url "https://github.com/koekeishiya/chunkwm/archive/v0.2.28.tar.gz"
+  sha256 "917c6aa09521ff4b7463f731192eea6be6313f0fa4e585f63f85603d0993cf92"
+
+  head do
+    url "https://github.com/koekeishiya/chunkwm.git"
+  end
+
+  option "without-tiling", "Do not build tiling plugin."
```

Comment:
I personally run chunkwm without the border plugin, and some might choose to use it only for the focus-follows-mouse. 

The transparency plugin requires an osax scripting bridge, so it's off by default as it requires a lot of additional work for the user.